opencode-test-carousel-animation_plan.md

```
@@ -0,0 +1,32 @@
+# Test Plan: Carousel Auto-Advance Animation
+
+## Context
```

markdownlint-cli2 (0.20.0)

[warning] Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)